api/graphql/schema.graphql

```diff
@@ -1184,6 +1184,48 @@ input GenericJobTitleBelongsToMany {
     sync: [ID!]
 }
 
+input CreatePoolAdvertisementInput {
+    key: KeyString!
+    classifications: ClassificationBelongsToMany!
+}
+
+input UpdatePoolAdvertisementInput {
+    # Pool name and classification
+    name: LocalizedStringInput
+    classifications: ClassificationBelongsToMany!
+    # Closing date
+    expiryDate: DateTimeTz @rename(attribute: "expiry_date")
```

Comment:
This seems to be shifting the time to my local time zone.  For example, this:
```
{  
	"id": "389b2be0-0f2c-425d-8f99-eb2b4ff861fd",
  "poolAdvertisement": {
    "expiryDate": "2011-12-03T10:15:30+01:00",
    "classifications": { }
  }
}
```
Results in this:

![image](https://user-images.githubusercontent.com/8978655/180056221-830a3cd8-2be1-4983-a28e-9744ed7d09a4.png)

I'd be OK if you wanted to kick this down the road to https://github.com/GCTC-NTGC/gc-digital-talent/issues/3315, too.

Reply:
Hmm, ya I'd like to get this merged in sooner then later since it's blocking other issues. Let's pass it onto #3315.